### ISSUE TITLE
[Feature] Keyboard shortcuts in inspector

### ISSUE THREAD

**Issue by huy-nguyen** [microsoft/playwright#12014]
### Discussed in https://github.com/microsoft/playwright/discussions/11891

<div type='discussions-op-text'>

<sup>Originally posted by **huy-nguyen** February  6, 2022</sup>
Is it possible to add keyboard shortcuts for debugging functionality in the inspector in a similar manner to Chrome dev tools, for example, F8 for run, F10 for step over etc?</div>

**Comment by viraxslot** [comment]
I'd like to implement it

**Comment by THELEMITED** [comment]
Yes, you might be able to ghost shell somebody with dagger@google and then deploy a an escaping method to parallel their circuits with embedded flutter code by google.

**Comment by navid4u** [comment]
I guess you've got to try ClojureScript Data Browser on chrome.

**Comment by jfgreffier** [comment]
Working on it

**Comment by canbax** [comment]
Is there any keyboard shortcut for "Explore"? 
![Screenshot 2023-02-18 at 10 53 11](https://user-images.githubusercontent.com/8426741/219848757-eb22ff8a-b6a7-4fc5-8a41-224811718469.png)

When I click on that the focus on my page goes away.

**Comment by jfgreffier** [comment]
@canbax can you ask the question on Discord, or create a new issue ?

**Comment by canbax** [comment]
@jfgreffier I created https://github.com/microsoft/playwright/issues/21249
